calicoctl/calico_ctl/node.py

```diff
@@ -340,6 +342,9 @@ def _start_node_container_docker(ip, ip6, as_num, log_dir, node_image, detach, e
     calico_networking = os.getenv(CALICO_NETWORKING_ENV,
                                   CALICO_NETWORKING_DEFAULT)
 
+    calico_gobgp = os.getenv(CALICO_GOBGP_ENV,
+                             CALICO_GOBGP_DEFAULT)
+
```

Comment:
We've talked about doing something similar to what you suggest, but haven't gotten around to it.

I think it would be nice to have something like you suggest, deprecating CALICO_NETWORKING in the process.  Need to think about the naming though.